include/deal.II/lac/ginkgo_solver.h

```diff
@@ -215,15 +224,341 @@ namespace GinkgoWrappers
     /**
      * Constructor.
      *
-     * @p solver_control The solver control object is then used to set the
-     * parameters and setup the CG solver from the CG factory which solves the
-     * linear system.
+     * @p[in,out] solver_control The solver control object is then used to set
+     * the parameters and setup the CG solver from the CG factory which solves
+     * the linear system.
+     *
+     * @p[in] exec_type The execution paradigm for the CG solver.
```

Comment:
That's still not correct. `@p` is a markup command, but you want `@param`. See http://www.doxygen.nl/manual/commands.html

Reply:
Sorry. My fault.